packages/logo/README.md

```diff
@@ -0,0 +1,65 @@
+# Logo
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/logo.svg)
+
+## Example
+
+```Javascript
+import { Logo } from '@leafygreen-ui/logo'
+
+<Logo />
+```
+
+**Output HTML**
+
+```HTML
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1112.61 300" height="40" width="100%">
+  <title>MongoDB Logo</title>
+  <path class="leafygreen-ui-17842x1" d="M134.44,120.34C119.13,52.8,87.22,34.82,79.08,22.11a144.57,144.57,0,0,1-8.9-17.42c-.43,6-1.22,9.78-6.32,14.33C53.62,28.15,10.13,63.59,6.47,140.33c-3.41,71.55,52.6,115.67,60,120.23,5.69,2.8,12.62.06,16-2.51,27-18.53,63.89-67.93,52-137.71"></path>
```

Comment:
Let's edit out all the path attribute values.

```
d="..."
```

Reply:
👍